docs/getting_started/troubleshooting.rst

```diff
@@ -11,7 +11,22 @@ Frequently encountered issues
 This is a list of the most frequently encountered issues when using Modin. Some of these
 are working as intended, while others are known bugs that are being actively worked on.
 
-Error During execution: ``ArrowIOError: Broken Pipe``
+
+Warning during execution: ``defaulting to pandas``
+"""""""""""""""""""""""""""""""""""""""""""""""""""
+Please note, that while Modin covers a large portion of the pandas API, not all functionality is implemented. For methods that are not yet implemented, such as ``asfreq``, you may see the following:
+
+.. code-block:: text
+
+  UserWarning: `DataFrame.asfreq` defaulting to pandas implementation.
+
+To understand which function will lead to this error, we have compiled a list of :doc:`currently supported methods </supported_apis/index>`. When you see this error, Modin defaults to pandas by converting the Modin dataframe to pandas to perform the operation. Once the operation is complete in pandas, it is coverted back to a Modin dataframe. These operations will have a high overhead due to the communication involved and will take longer than pandas. When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. You can learn more about this :doc:`here <supported_apis/defaulting_to_pandas>`.
```

Comment:
I would change all the "errors" here to "warning" -- i.e. "when you see this warning, Modin..."

Reply:
Changed all errors to warning here.